src/components/VariablesTable.vue

```diff
@@ -148,18 +166,22 @@
   const variablesCountSubscription = useSubscription(api.variables.getVariablesCount, [filter])
   const variablesCount = computed(() => variablesCountSubscription.response)
 
-  function refresh(): void {
+  function refreshSubscriptions(): void {
     variablesSubscription.refresh()
     variablesCountSubscription.refresh()
   }
 
+  defineExpose({
+    refreshSubscriptions,
+  })
+
```

Comment:
I don't think we're doing this in a ton of places so I wanted to call it out - this allows us to refresh the subscription when an external event renders it necessary - in this case it allows us to refresh when a new variable is added from elsewhere

Reply:
Also I made the method name more verbose to reflect its use externally - `refresh` didn't seem clear enough when calling it from elsewhere

Reply:
This is something that the subscription pattern doesn't handle well. Don't think I have a better idea. Usually we just recreate the subscription externally but the filters here are so specific you wouldn't be able to.

Reply:
Agreed - honestly this doesn't feel that bad component-side; it's a little odd consumer-side cause you're creating a `ref`, but it's well-typed so it's easy to inspect